src/topology/order.lean

```diff
@@ -189,6 +208,16 @@ generate_from_le_iff_subset_is_open
 instance : complete_lattice (topological_space α) :=
 @order_dual.complete_lattice _ tmp_complete_lattice
 
+lemma is_open_implies_is_open_iff {a b : topological_space α} :
+  (∀ s, a.is_open s → b.is_open s) ↔ b ≤ a :=
+@galois_insertion.u_le_u_iff _ _ _ _ _
+  (@partial_order.to_preorder _
+    (@semilattice_sup.to_partial_order _
+      (@lattice.to_semilattice_sup _
+        (@bounded_lattice.to_lattice _
+          (@complete_lattice.to_bounded_lattice _ tmp_complete_lattice)))))
+  (gi_generate_from α) a b
```

Comment:
```suggestion
@galois_insertion.u_le_u_iff _ (order_dual (topological_space α)) _ _ _ _ (gi_generate_from α) a b
```
Hopefully the same works below?

Reply:
Thanks, yes!  I've changed throughout.  I'm suprised because `order_dual (topological_space α)` actually carries the dual of the dual of `tmp_order`, whereas the Galois insertion is stated for `tmp_order`.  I had assumed that an order was not defeq to its double dual.